ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                               አሐ</hi>ዱ፡ አምላክ፨ ንወጥን፡ በፈሪሀ፡ እግዚአብሔር፨ <hi rend="rubric">ሎቱ፡ ስብሐተ፡ በእንተ፡ ጽሒፈ፡ ሰዐታተ፡ ወጸሎታ</hi>ተ፡
+                               ዘመዓልተ፡ ወለሊተ፡
+                               ወውእቱ፡ ረዳኢነ፡ ምስለ፡ ገቢ<hi rend="rubric">ር፡ ለፈጽሞ፡ ዘነገሮ፡ መልአከ፡ እግዚአብሔር፡ ለ</hi>አባ፡ እንጦንስ፡
+                             ጸሎቱ፡ ተሀሎ፡ ምስሌነ፡ አሜን፨ </incipit>
+                            <explicit xml:lang="gez">ይስረይ᎓ ኃጢአተነ᎓ በብዝኃ᎓ ምሕረቱ᎓ ሰአሊ᎓ ለነ᎓ ቅድስት፨፨፨</explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.5">
+                             <locus from="157r" to="181v"/>
+                             <title type="complete" ref="LIT2509Weddas"/>
+                             <textLang mainLang="gez"/>
+                             <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። አ</hi>
+                             ሐዱ᎓ አምላክ፨ ወእምዝ፡ ንጽሕፍ፡ ው<hi rend="rubric">ዳሴሃ᎓ ለእግዚእትነ᎓ ማርያም᎓ ድንግል፡ ወላ</hi>ዲተ᎓ አምላክ᎓ <gap reason="ellipsis" resp="DN"/>
+                             ተሰመይኪ᎓ ፍቅርተ፡ ኦቡርክት፡ እምአንስት፡ አንቲ፡ ይእቲ፡ ዳግሚት፡ ቀመር፡ እንተ፡ ተሰመይኪ፡ ቅድስተ፡ ቅዱሳን። ዘውስቴታ፡ ጽላተ፡ ኪዳን፡</incipit>
+                             <explicit xml:lang="gez">
+                             ኦእግዚእትነ᎓ እሙ᎓ ለዐቢይ፡ ወክቡር፡ አሐዜ፡ ኵሉ፡ ዓለም፡ ዘምሉእ፡ ሰማያ፡ ተ፡ ወምድር፡ ስብሐቲሁ፡ ሎቱ፡ ንገኒ፡ ወሎቱ፡ ነአምን፡
+                             እምይእዜ፡ ወእስከ፡ ለዓለም፡ ዓለም፡ አሜን፨ <hi rend="rubric">
+                             ሠረቀ᎓ በሥጋ᎓ እምድንግል᎓ ዘእን</hi></explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.6">
+                             <locus from="181v" to="186ra"/>
+                             <title type="complete" ref="LIT4711Mastabqwe"/>
+                             <textLang mainLang="gez"/>
+                               <incipit xml:lang="gez"><hi rend="rubric">ውዳሴ᎓ ወግናይ᎓ ዘእግዚእትነ᎓ ማርያም᎓᎓</hi> ወላዲተ᎓
+                             አምላክ፨ ጸሎታ᎓ ወበረከታ᎓ የሀሉ᎓ ምስለ᎓ ገብራ᎓ <gap reason="lost"/>ለዓለመ᎓ ዓለም᎓ አሜን፨
+                             <hi rend="rubric">ቅድስት፡ ወብፅዕ</hi>ት᎓ ስብሕት᎓ ወቡርክት፨ ክብርት᎓ ወልዕልት፨ አንቀጸ᎓ ብርሃን᎓
+                             መዐርገ᎓ ሕይወት᎓</incipit>
+
+                             <msItem xml:id="ms_i1.6.1">
+                               <locus from="181v" to="18vb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዱያን፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ኀዋጺሃ፡ ለነፍስ፡ ወመፈውሰ፡ ሥጋ፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.2">
+                               <locus from="181vb" to="182rb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ እለ፡ ይነግድ፡</hi></title>
```

Comment:
no textpart yet under LIT4711Mastabqwe? should be introduced? or is it within SupplicationSick?

Reply:
yes, it is there

Reply:
Under "4711" it is described as SupplicationTraders, it is for those who travel. I will correct in the work record